addons/spectator/CfgVehicles.hpp

```diff
@@ -16,65 +16,49 @@ class CfgVehicles
     };
     class GVAR(module) : Module_F
     {
-        // Standard object definitions
-        scope = 2; // Editor visibility; 2 will show it in the menu, 1 will hide it.
-        displayName = "Spectator: Show Objective"; // Name displayed in the menu
-
+        scope = 2;
+        displayName = "Spectator: Show Objective";
         category = "Teamwork";
         icon = "\x\tmf\addons\common\UI\logo_tmf_small_ca.paa";
-
-        // Name of function triggered once conditions are met
         function = QFUNC(objectiveModule);
-        // Execution priority, modules with lower number are executed first. 0 is used when the attribute is undefined
         functionPriority = 0;
-        // 0 for server only execution, 1 for global execution, 2 for persistent global execution
         isGlobal = 2;
-        // 1 for module waiting until all synced triggers are activated
         isTriggerActivated = 1;
-        // 1 if modules is to be disabled once it's activated (i.e., repeated trigger activation won't work)
         isDisposable = 0; // broken in EDEN;
-
-        // Menu displayed when the module is placed or double-clicked on by Zeus
-
-
-        // Module arguments
         class Arguments: ArgumentsBaseUnits
```

Comment:
Perhaps change to 3DEN Attributes instead of legacy Arguments?

Reply:
Out of scope here I think, something to do in the future for sure.